### ISSUE TITLE
--fix only fixes variable declaration not usages of the variable

### ISSUE THREAD

**Issue by DeveloperDanjo** [erikdesjardins/eslint-plugin-dollar-sign#7]
Say I have the following code:

```
var someSelection = $('#foo');
someSelection.find('p').text('blah');
```

The `--fix` changes this to:

```
var $someSelection = $('#foo');
someSelection.find('p').text('blah');
```

I would expect it to change all usages, such as

```
var $someSelection = $('#foo');
$someSelection.find('p').text('blah');
```

**Comment by DeveloperDanjo** [comment]
I am not familiar with the internal's provided by eslint for doing fixes, so if this is not possible I would imagine that disabling the fix mechanism all together would be safer than making an incorrect fix to the checked code.

**Comment by erikdesjardins** [comment]
Sorry about the delay.

ESLint's fixing mechanism does make this hard (you can only change one token per error reported), but it should be feasible by reporting an error for each var reference.

I'll remove the autofixing for object props, though, because that's definitely going to break things:

``` js
function foo() {
  var obj = { x: $('.foo') };
  bar(obj);
}

function bar(obj) {
  doSomething(obj.x); // no feasible way to change this to `$x`
}
```

**Comment by erikdesjardins** [comment]
Released as v1.0.0, thanks!
